testSuite.py

```diff
@@ -0,0 +1,113 @@
+from utilities import ReturnCode
+
+# timeout for pexpect. short_timeout for the cases where there should not be communication
+# and long_timeout for the cases where there should be
+short_timeout = 5
+long_timeout = 20
+
+
+ReturnCode.export_to(globals())
+
+dict_param_expected_code_timeout = {
```

Comment:
this name is kind of weird, how about `test_suite` ? or `rtps_test_suite_1` (to avoid conflicts whenever we have the option to add more test suites

Reply:
yes, i think it is better